iidm/iidm-xml-converter/src/main/resources/xsd/iidm.xsd

```diff
@@ -260,9 +260,15 @@
     <xs:complexType name="ShuntCompensator">
         <xs:complexContent>
             <xs:extension base="iidm:Injection">
+                <xs:sequence>
+                    <xs:element name="regulatingTerminal" minOccurs="0" type="iidm:TerminalRef"/>
+                </xs:sequence>
                 <xs:attribute name="bPerSection" use="required" type="xs:double"/>
                 <xs:attribute name="maximumSectionCount" use="required" type="xs:int"/>
                 <xs:attribute name="currentSectionCount" use="required" type="xs:int"/>
+                <xs:attribute name="regulating" use="optional" type="xs:boolean"/>
+                <xs:attribute name="targetV" use="optional" type="xs:double"/>
+                <xs:attribute name="targetDeadband" use="optional" type="xs:double"/>
```

Comment:
Are all of these attributes really optional or do you set them optional for backward compatibility?

Reply:
`targetV` and `targetDeadband` are really optional (they can be undefined if `regulating` is `false`). for `regulating`, I can put it as required but it means that all previous files would not be readable.